bacchus_gazebo/launch/vineyard_demo.launch

```diff
@@ -21,17 +21,10 @@
   <arg name="use_omni" default="false"/>
   <arg name="world_2_map_pose" default="0.0 0.0 0.0 0.0 0.0 0.0 1.0" />
   
-  <arg name="X_INIT" value="40.0" if="$(arg riseholme_dt)"/>
-  <arg name="Y_INIT" value="-40.0" if="$(arg riseholme_dt)"/>
-  <arg name="Z_INIT" value="1.2" if="$(arg riseholme_dt)"/>
-  <arg name="loc_map_yaml" value="$(find bacchus_gazebo)/maps/riseholme.yaml" if="$(arg riseholme_dt)"/>
-  <arg name="nav_map_yaml" value="$(find bacchus_gazebo)/maps/riseholme.yaml" if="$(arg riseholme_dt)"/>
-
-  <arg name="X_INIT" value="-11.0" unless="$(arg riseholme_dt)"/>
-  <arg name="Y_INIT" value="0.0" unless="$(arg riseholme_dt)"/>
-  <arg name="Z_INIT" value="1.0" unless="$(arg riseholme_dt)"/>
-  <arg name="loc_map_yaml" value="$(find bacchus_gazebo)/maps/cropped.yaml" unless="$(arg riseholme_dt)"/>
-  <arg name="nav_map_yaml" value="$(find bacchus_gazebo)/maps/cropped.yaml" unless="$(arg riseholme_dt)"/>
```

Comment:
Not sure we want to remove this,  leaving the flag `riseholme_dt`. It will create confusion. I suggest to leave as it is or remove every reference to the digital twin environment.